entities-viewings-collector/src/test/scala/io/renku/entities/viewings/search/DatasetQuerySpec.scala

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2023 Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.renku.entities.viewings.search
+
+import io.renku.graph.model.entities
+import io.renku.entities.search.model.{MatchingScore, Entity => SearchEntity}
+import io.renku.entities.viewings.search.RecentEntitiesFinder.Criteria
+import io.renku.generators.Generators.Implicits._
+import io.renku.http.server.security.model.AuthUser
+
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+
+class DatasetQuerySpec extends SearchTestBase {
+
+  it should "return multiple datasets" in {
+    val project1 = renkuProjectEntities(visibilityPublic)
+      .withActivities(activityEntities(stepPlanEntities()))
```

Comment:
Do we need activities for these specs?

Reply:
right, probably not needed! I'll remove them